buildSrc/subprojects/performance-testing/src/main/kotlin/gradlebuild/performance/PerformanceTestPlugin.kt

```diff
@@ -269,34 +270,36 @@ class PerformanceTestPlugin : Plugin<Project> {
             createDistributedPerformanceTestTask(name, clazz, performanceSourceSet, prepareSamplesTask).configure(configure)
         }
 
+        val channelSuffix = if (OperatingSystem.current().isLinux) "" else "-windows"
```

Comment:
Can we use something like `OperatingSystem.current().name.toLowerCase()` or something so that this works for macOS as well?